app/src/main/java/com/abhijitvalluri/android/fitnotifications/services/NLService.java

```diff
@@ -88,6 +92,7 @@ public void onCreate() {
         SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(this);
         mIsServiceEnabled = true;
 
+        // FIXME: preferences keys should not be "translatable" ?
```

Comment:
`strings.xml` is used for "localizable" text, visible to the user. If someone translates this text to German in `strings_de.xml` then the user will loose his settings after switching the language.
These are internal preferences keys. And they should be declared as internal constants instead.

Besides, there is computational overhead here. To retrieve a preference you cause loading of the resource bundle and resolving an `int` into corresponding text.